packages/governance/src/types-ambient.js

```diff
@@ -49,7 +49,7 @@
  * T extends 'ratio' ? Ratio :
  * T extends 'string' ? string :
  * T extends 'timestamp' ? import('@agoric/time/src/types').TimestampRecord :
- * T extends 'relativeTime' ? import('@agoric/time/src/types').RelativeTimeRecord :
+ * T extends 'relativeTime' ? import('@agoric/time/src/types').RelativeTimeRecord | import('@agoric/time/src/types').RelativeTime :
```

Comment:
no need for the deep import, and the union is still the default. 
```suggestion
 * T extends 'timestamp' ? import('@agoric/time').TimestampRecord :
 * T extends 'relativeTime' ? import('@agoric/time').RelativeTime :
```

should `TimestampRecord` also accept an unbranded time?

Reply:
Thanks. Timestamp and RelativeTime each include both appropriate types.

The first time through here I didn't notice the curious syntax `T extends 'name' ? type :`.  This time it slowed me down considerably.